docs/concepts/01-what-is-sablier.mdx

```diff
@@ -13,61 +13,52 @@ import ReleaseHistoryLockup from "@site/docs/snippets/ReleaseHistoryLockup.mdx";
 from one account to another, in realtime.
 
-- **The Sablier Protocols**: A collection of persistent, non-upgradeable smart contracts to facilitate streaming of
-  ERC-20 tokens on Ethereum and other EVM blockchains. The Sablier Protocols consist of Lockup, Merkle Airdrops and
+- **The Sablier Protocol**: A collection of persistent, non-upgradeable smart contracts to facilitate streaming of
+  ERC-20 tokens on Ethereum and other EVM blockchains. The Sablier Protocol consists of Lockup, Merkle Airdrops, and
   Flow.
-- **The Sablier Interface**: A web interface that allows for easy interaction with the Sablier Protocols. The interface
-  is only one of many ways to interact with the Sablier Protocols.
-- **Sablier Labs**: The company that develops the Sablier Protocols, the Sablier Interface, and the documentation
-  website you are reading right now.
+- **The Sablier Interface**: A web interface that allows for easy interaction with the Sablier Protocol. The interface
+  is only one of many ways to interact with the Sablier Protocol.
+- **Sablier Labs**: The company that develops the Sablier Protocol, the Sablier Interface, and the documentation website
+  you are reading right now.
 
 :::info
 
 Fun fact: "sablier" means "hourglass" in French.
 
 :::
 
-## Sablier Protocols
+## Sablier Protocol
 
-Sablier protocols are a collection of token distribution protocols developed with [Ethereum](https://ethereum.org/)
-smart contracts, designed to facilitate by-the-second payments for cryptocurrencies, specifically
-[ERC-20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) tokens. The protocols employ a set of
+A software protocol built with [Ethereum](https://ethereum.org/) smart contracts, designed to facilitate distribution of
+[ERC-20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) tokens. The protocol employs a set of
 persistent and non-upgradable smart contracts that prioritize security, censorship resistance, self-custody, and
 functionality without the need for trusted intermediaries who may selectively restrict access.
 
-Currently, it consists of three protocols: **Lockup** to facilitates vesting and airstreams, **Merkle Airdrops** to
-enable on-chain airdrops and **Flow** for payroll, grants etc. All of these are licensed under BUSL-1.1, and are
-open-source and can be accessed on Sablier's [GitHub page](https://github.com/sablier-labs). Detailed technical
-reference for each protocol can be found in the [Technical References](/reference/overview) section of this website.
+Currently, Sablier consists of three separate systems:
 
-As long as Ethereum and the other EVM chains continue to exist, every version of Sablier protocols that gets deployed
+- **Lockup**: facilitates vesting and vested airdrops
+- **Merkle Airdrops**: enables on-chain airdrops
+- **Flow**: for payroll, grants etc.
+
+All of these are licensed under BUSL-1.1 and GPL v3, and the source code can be accessed on Sablier's
```

Comment:
Saying that "All of these are licensed under BUSL-1.1 and GPL v3" would mean that both license apply to each. How about:

While most of these are licensed under BUSL-1.1, there are some components licensed under GPL v3

Reply:
Good suggestion